src/definitions/towns.ts

```diff
@@ -4,7 +4,7 @@ import { EcoChain } from '../chain.interface'
 export const townsSepolia = /*#__PURE__*/ defineChain({
   id: 6524490,
   name: 'Towns Sepolia',
-  nativeCurrency: { name: 'tDMT', symbol: 'tDMT', decimals: 18 },
```

Comment:
I think this is not correct, looking at this transaction the native currency is ETH.

https://towns-devnet.explorer.caldera.xyz/tx/0x1bea5b397ebee130d642b6e271cb4e02aac157c0d33178127dc971fc80adf0e6